src/main/webapp/app/pages/curation/collapsible/ReviewCollapsible.tsx

```diff
@@ -239,21 +239,29 @@ export const ReviewCollapsible = ({
     if (rootReview.reviewLevelType === ReviewLevelType.META) {
       return undefined;
     }
+    const deletionAction = (
+      <ActionIcon
+        icon={faTimes}
+        color={DANGER}
+        onClick={() => {
+          handleActionClick(ActionType.REJECT);
+        }}
+        disabled={disableActions}
+      />
+    );
     if (isCreateReview(rootReview)) {
-      // Reviewable content should be reviewed individually under a create event
-      return undefined;
+      if (rootReview.nestedUnderCreateOrDelete) {
```

Comment:
Just wondering why we don't allow deleting a nested creation. For example, when I create a mutation and then a cancer type under it, why do we not allow rejecting the entire cancer type?

Reply:
Yes, I think nested creations should be deletable as well. However, let's consider the scenario you described. If we reject the cancer type creation (which is the only nested review under the mutation), then the mutation gets automatically accepted. This is due to the behavior we implemented that a created entity will be auto-accepted when all items have been reviewed. 

We have a ticket to add a `isReviewPending` status to review object, so that we can keep track of the items that are pending acceptance.

Reply:
That's a good point. Thanks for letting me know.